eslint.config.mjs

```diff
@@ -12,64 +12,66 @@ import nPlugin from 'eslint-plugin-n';
 const filename = fileURLToPath(import.meta.url);
 const dirname = path.dirname(filename);
 
-export default defineConfig(
```

Comment:
@brijeshb42 Not sure why but typescript started complaining about non-portable types again

Reply:
Opening https://github.com/eslint/eslint/issues/20189